stories/topics/proc-azure-monitor-deployment-engine.adoc

```diff
@@ -0,0 +1,21 @@
+[id="azure-monitor-deployment-engine_{context}"]
+
+= Monitoring deployments on the {PlatformNameShort} Deployment Engine
+
+[role="_abstract"]
+
+The deployment engine displays information about your {AAPonAzureNameShort} deployment.
+You can monitor the progress of the deployment and you can cancel the deployment.
+
+When you begin deploying {AAPonAzureNameShort}, the Azure interface displays the *Overview* page for the deployment.
+
+. Click *Parameters and Outputs* in the *Settings* menu.
++
+Approximately 10 minutes into the deployment process, the *Outputs* section of the *Parameters and Outputs* page displays a link to the *`deploymentEngineUrl`*.
+. Copy the link and paste it in another browser tab to open the login page for the deployment engine.
+. Login to the deployment using the following credentials:
+  * *Username*: _admin_
+  * *Password*: Use the _Administrator Password_ that you chose when configuring your deployment.
+
+The {PlatformNameShort} Deployment Engine displays a list of the steps in the deployment process. Icons indicate the steps that have been completed and the steps that are in progress.
```

Comment:
Icons also indicate the deployment steps which are not started to run yet.

Reply:
@ariordan-redhat , you are right. There is no icon for steps that have not begun. You can close my comment on it.